projects/novo-elements/src/elements/form/extras/address/Address.ts

```diff
@@ -273,7 +273,7 @@ export class NovoAddressElement implements ControlValueAccessor, OnInit {
         }
         this.stateOptions = this.config[field].pickerConfig.options;
         this.config[field].pickerConfig.options = (query = '') => {
-          return this.stateOptions(query, this.model.countryID);
+          return this.stateOptions(query, undefined, this.model.countryID);
```

Comment:
If you were to change the stateOptions in Novo to have page be the third parameter, you would not need to force a change on novo elements.